Widgets/Controls/RenderTime.ascx.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Widgets.Controls {
+    public partial class RenderTime : System.Web.UI.UserControl {
+        public string format { get; set; }
+        public string message { get; set; }
+
+        protected void Page_Load(object sender, EventArgs e) {
+            //format = "M/d/yyyy h:mm tt";
+            DateTime dateTime = DateTime.Now;
+            message += "Rendered at: ";
+            message += dateTime.ToString(format);
+            //message = message + Convert.ToDateTime(dateTime).ToString(format);
+            Message.Text = message;
+        }
```

Comment:
Just as an FYI, the goal for this user control was to provide `Label` and `Format` properties to allow the "Rendered at: " text and the date/time format to be customized. If either of those properties doesn't have a value, default values should be used.